tests/cpp/test_learner.cc

```diff
@@ -92,4 +93,26 @@ TEST(Learner, CheckGroup) {
   delete pp_mat;
 }
 
+TEST(Learner, SLOW_CheckMultiBatch) {
```

Comment:
Should we add a filter for `SLOW` in C++ tests?

Reply:
@hcho3 It's OK for the test farm,  just not so convenient during development.